jest-opposite-of-matcher/isNotEqual.js

```diff
@@ -0,0 +1,5 @@
+const isNotEqul = () => {
+  for (let a = 1; a < 10; a++) {
+    return a + b;
+  }
+};
```

Comment:
I'm not sure what this method is supposed to be doing either. It looks like it will always return `1 + b`.

Reply:
I originally coded this function like this. Maybe this is more correct?

```javascript
const isNotEqual = () => {
  for (let a = 1; a < 10; a++) {
    for (let b = 1; b < 10; b++) {
      return a + b;
    }
  };
```

Then I second-guessed myself, thinking that I was just copying the outline of **isNotEqual.test.js** Because I got the same result with both of these iterations, I thought I was being more succinct.

On second glance, after your note, I think—correct me if I'm wrong—that the problem lies in the fact that in the code you query, all operation would cease after the `return` statement, whereas when the return statement is encapsulated within an inner loop as above, it would iterate all the way from 1-10 on the outer loop.

Am I onto something here, or am I misunderstand how this works?